src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -324,7 +324,7 @@ nuget pack foo.nuspec -Version 2.1.0</value>
     <value>Pushes a package to the server and publishes it.</value>
   </data>
   <data name="PushCommandSourceDescription" xml:space="preserve">
-    <value>Specifies the server URL. If not specified, nuget.org is used unless DefaultPushSource config value is set in the NuGet config file.</value>
+    <value>Specifies the package source the package will be pushed to. This is a mandatory parameter unless the NuGet.Config file specifies a DefaultPushSource value.</value>
     <comment>Please don't localize "DefaultPushSource"</comment>
   </data>
   <data name="PushCommandSymbolSourceDescription" xml:space="preserve">
```

Comment:
I'm not confident that nuget.smbsrc.net is de the default source when you push to nuget.org a package with symbols. For example:

```
 nuget push .\pkg.1.0.0.nupkg -source nuget.org -apikey xyz
Pushing pkg.1.0.0.nupkg to 'https://www.nuget.org/api/v2/package'...
  PUT https://www.nuget.org/api/v2/package/
  Created https://www.nuget.org/api/v2/package/ 886ms
Your package was pushed.
Pushing pkg.1.0.0.snupkg to 'https://www.nuget.org/api/v2/symbolpackage'...
  PUT https://www.nuget.org/api/v2/symbolpackage/
  Created https://www.nuget.org/api/v2/symbolpackage/ 397ms
Your package was pushed.
```

I prefer to have a description like the following:

```
Symbol Server URL to be pushed to.
```

Reply:
I assume this is referring to `PushCommandSymbolSourceDescription`? I suggested to change those [above](https://github.com/NuGet/NuGet.Client/pull/2744#issuecomment-544710719) but I was told that it won’t matter.

Reply:
Yes, your assumption is correct. Thanks for pointing out that. 

The case above was for a package with .snupkg symbols format.

For legacy .symbols.nupkg format,, there's no automatic symbols publishing to nuget.smbsrc.net: 
```
nuget push .\pkg.0.0.1.nupkg -verbosity detailed
NuGet Version: 5.4.0.6315
Pushing pkg.0.0.1.nupkg to 'https://www.nuget.org/api/v2/package'...
  PUT https://www.nuget.org/api/v2/package/
  Created https://www.nuget.org/api/v2/package/ 789ms
Your package was pushed.
```